voyager_archive/templates/voyager_archive/marc_display.html

```diff
@@ -11,18 +11,31 @@
 </form>
 <hr/>
 
-{% comment %}
-Raw Marc dump
-{% endcomment %}
+<div class="row">
+    <div class="column">
+        <h4>Record Leader: {{ marc_record.leader }}</h4>
+        <table class="data-display">
+            {% for field in marc_record.fields %}
+            <tr>
+                <td>{{ field.tag }}</td>
+                <td><span class='indicators'>
+                    {{ field.indicator1|default:"_" }}
```

Comment:
Control fields (001-009) do not have indicators, so displaying `_ _` is not ideal.  With the blank -> underscore replacement done in `get_marc_fields()` (see other comment), the defaults are no longer needed here, so just
```
{{ field.indicator1 }}
{{ field.indicator2 }}
```